cron/client.py

```
@@ -63,6 +63,19 @@ def set_status(status_code, data=None, run_id=None):
 
+def reboot_if_uptime_exceeded(reboot_after_s):
+
+    if not reboot_after_s:
+        return
+
+    with open('/proc/uptime', encoding='UTF-8') as f:
+        uptime_seconds = float(f.read().split()[0])
+
+    if uptime_seconds > reboot_after_s:
+        print(f"Uptime {uptime_seconds:.0f}s exceeds reboot_after_seconds {reboot_after_s}s. Rebooting...")
+        subprocess.check_output(['sync'], encoding='UTF-8', errors='replace')
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)